site/static/api/device-registry-v1.yaml

```diff
@@ -0,0 +1,724 @@
+openapi: 3.0.1
+info:
+   title: Eclipse Hono™ Device Registry API
+   description: |
+      This API allows one to interact with the Eclipse Hono Device Registry
+      API. It acts as a common basis which all Hono device registries should
+      implement.
+
+      ## Required APIs
+
+      All operations, except the `tenants` resource are required. The tenant
+      management might be outside of the scope of the device registry and
+      managed by a higher level system. In this case all calls should simply
+      return `404`. However, if the `tenants` resource is implemented, then all
+      operations of it must be implemented.
+
+      ## Security
+
+      This specification explicitly leaves out the part of authenticating and
+      authorizing users with the device registry. It is assumed that some form
+      of token exchange between the user agent and the backend service will
+      take place. Like for example HTTP basic authentication, or a bearer token.
+
+      ## Code generation
+
+      This model is not optimized for generating code from it. Code generators
+      try to understand the model and then translate this into the require
+      programming language. Even if the there would be no bugs in the code
+      generators, that process would be already only be an approximation. So
+      this model does focus in the description of the API, and doesn't tweak
+      the specification in a way to please code generators.
+
+   contact:
+      email: "hono-dev@eclipse.org"
```

Comment:
this seems a little problematic, as you can only send to this address, if you have subscribed to the list ...

Reply:
I am open to suggestions.

Reply:
I would replace the *email* property with a *url* property pointing to https://www.eclipse.org/hono/community/get-in-touch/

Reply:
Done.